JavaCryptographicArchitecture/pom.xml

```diff
@@ -4,7 +4,7 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>de.darmstadt.tu.crossing</groupId>
 	<artifactId>JavaCryptographicArchitecture</artifactId>
-	<version>1.5.0</version>
+	<version>1.5.2</version>
```

Comment:
Why jump from 1.5.0 to 1.5.2 instead of 1.5.1?

Reply:
I was checking the Nexus Repository where all the rulesets are hosted and noticed that the latest JCA version pushed there was 1.5.1 and I am unsure why we were still using 1.5.0. Anyway, I can change the version here at 1.5.1 and the repository version will be updated with the newest changes.

Reply:
No, let's bump to version 1.5.2. Can you check for any changes that might be missed from version 1.5.1?

Reply:
I just remembered now why is there a 1.5.1 version. It was me who bumped the version in order to do some testing with the failing build of CryptoAnalysis around three months ago. The ruleset with 1.5.1 is just redundant and I had forgot to delete it in the Nexus server. I, therefore, think that we instead use version 1.5.1 that includes the changes in this PR.